.github/workflows/publish.yaml

```
@@ -19,6 +19,8 @@ on:
     branches-ignore:
       - "dependabot/**"
       - "pre-commit-ci-update-config"
+  tags:
+    - "**"
```

```suggestion
    tags:
      - "**"
```
Doesn't this need to be indented a bit more to come under `push`?

---

Yes! Thanks!